### ISSUE TITLE
Feature Request: Default Admin name email

### ISSUE THREAD

**Issue by ghost** [Automattic/studio#87]
### What

I would love the option to change the default username, email and password for spining up new sites directly within the Studio application (a "preferences" setting maybe?)... directly on the local computer.

### Why

I don't know anyone who ever uses "admin" for an admin account. We all have our own information for this as best practices. This would save a step of having to go in and add my normal info in a new admin account, and then deleting the default one. It would just save everyone a few steps.

### How

Sorry I can't help with "How" - Only thing I can think of is that it could be something within the default settings or preferences (whatever term you are using for that option in the menu).

p.s. you might take a look at how Local does the site setup in order to do something similar. It allows you to set up a default admin setting OR change it right there when you enter the site name.

**Comment by paaljoachim** [comment]
I agree it would be very helpful to be able to add a username, email and password when setting up a site.
Here is how LocalWP does this.

<img width="753" height="469" alt="Image" src="https://github.com/user-attachments/assets/820735d7-96a1-41f5-b106-37968fc8e601" />

**Comment by wojtekn** [comment]
We've added that feature in Studio. The change is merged to the trunk and will be available in the next version.
